src/SimplCommerce.WebHost/Themes/CozaStore/Views/Shared/_Layout.cshtml

```diff
@@ -254,9 +256,9 @@
     <script simpl-append-version="true" src="~/lib/matchheight/jquery.matchHeight.js"></script>
     <script simpl-append-version="true" src="~/lib/toastr/build/toastr.min.js"></script>
     <script simpl-append-version="true" src="~/lib/bootstrap-star-rating/js/star-rating.js"></script>
-    @if (CultureInfo.CurrentCulture.Name == "vi-VN")
+    @if(CultureInfo.CurrentCulture.Name != "en-US")
     {
-        <script simpl-append-version="true" src="~/lib/bootstrap-star-rating/js/locales/vi-vn.js"></script>
+    <script simpl-append-version="true" src="~/lib/bootstrap-star-rating/js/locales/@(CultureInfo.CurrentCulture.Name).js"></script>    
```

Comment:
I was planned to do this in the past, please check if the star-rating using two letter culture for all the locales

Reply:
I will do this in the next PR.
I'll create a standard function for handling multi-language files

funtionX("~/lib/bootstrap-star-rating/js/locales/{0}.js")
....
return "~/lib/bootstrap-star-rating/js/locales/pt-BR.js"
OR
return "~/lib/bootstrap-star-rating/js/locales/pt.js"

depending on which file exists